.env.playwright

```diff
@@ -1 +1,23 @@
-BASE_URL=https://localhost:3000
\ No newline at end of file
+PROJECTS='{
```

Comment:
It looks like it will be more maintainable to save this JSON config to a JSON file and then read from it when the tests start.

Reply:
I tried to do it, but it seems like some issues appear than. We should better postpone until we know how to make a workable beautiful playwright config

Reply:
What kind of issues appeared?

Reply:
fixed